pkg/scheduler/api/resource_info.go

```diff
@@ -296,6 +296,34 @@ func (r *Resource) Multi(ratio float64) *Resource {
 	return r
 }
 
+// ScaleResourcesWithRatios multiples the resource with ratio provided for CPU, Memory, and ScalarResources
+func (r *Resource) ScaleResourcesWithRatios(ratio map[string]float64, defaultRatioKey string) *Resource {
```

Comment:
default should be a float.

Reply:
The default value itself is also a float, and the key of the default value is passed in.

Reply:
So there is no improvement now？

Reply:
It still has dependence on other context. This func should be clean.

Reply:
fixed